conf/application.conf

```diff
@@ -133,8 +133,8 @@ microservice {
     eis {
       protocol = "http"
       host = "localhost"
-      uri = "/common-transit-convention-trader-at-destination/message-notification"
-      port = 9481
+      uri = "/movements/messages"
```

Comment:
Is this right? Should this not have a prefix of _transit-movements-trader-eis-router_